app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/PictureStorage.kt

```diff
@@ -12,14 +12,25 @@ import java.io.File
 import java.io.IOException
 import java.util.*
 
-open class PictureRepository(
+/**
+ * Allows to access picture from and save them to local storage
+ * @property dbName name of the database the pictures belong to
+ * @property context application context
+ * @property folder can be used to specify storage location, by default directory named after$
+ * the database in local storage
+ */
+open class PictureStorage(
```

Comment:
```suggestion
class PictureCache(
```
1. "Storage" is not a countable singular noun, hence it is not suitable her. Instances of this class get called `cache` all over the codebase, hence as far as I am concerned it implements a cache.
2. #151 indicated that this class being open made little sense. Let's close it.

Reply:
I forgot about 2.
Cache is a better name indeed